physlr/physlr.py

```diff
@@ -2109,6 +2109,51 @@ def chr_int(x):
             return None
         return int(x[3:]) if x.startswith("chr") else int(x)
 
+    def physlr_annotate_graph_bxtochr(self):
+        """
+        Annotate a graph with a file of barcode to chromosome mappings.
+        The mapping file includes two values per line separated by tab: barcode an chromosome.
+        Usage: physlr physlr-annotate-graph-bxtochr GRAPH PATH MAPPINGS_FILE... >ANNOTATED-GRAPHVIZ
+        """
+
+        if len(self.args.FILES) != 2:
+            sys.exit("physlr annotate-graph-bxtochr: error: two file arguments are required")
+        graph_filenames = [self.args.FILES[0]]
+        map_filename = self.args.FILES[1]
+
+        g = self.read_graph(graph_filenames)
+        Physlr.remove_small_components(g, self.args.min_component_size)
+
+        # Read barcode to chromosome data into a dictionary
+        bxtochr = {}
+        with open(map_filename) as mapfilename:
+            for line in mapfilename.readlines():
+                bx_chr = line.split()
```

Comment:
```suggestion
                bx, chr = line.strip().split()
```
I always like to add a `strip` in there to get rid of any pesky white space. Also thinking you could unpack the values to the `bx` and `chr` variables, and then use them in the next line. Just nice for readability.